app/src/main/AndroidManifest.xml

```diff
@@ -172,6 +173,10 @@
         <activity android:name=".widgets.button.ButtonWidgetConfigureActivity"
             android:configChanges="orientation|screenSize"
             android:exported="true">
+        </activity>
+        <activity android:name=".widgets.common.WidgetAuthenticationActivity"
+            android:configChanges="orientation|screenSize"
+            android:exported="true">
```

Comment:
The intent filter for `android.appwidget.action.APPWIDGET_CONFIGURE` is now added to the authentication activity but it should be for the button widget configuration activity instead.

Reply:
True, I don't know how this happened - thanks for spotting.